.dockerignore

```
@@ -1,11 +1,6 @@
 node_modules
-.git
-examples
 **/node_modules
+!packages/plugin-node-surrounding-code/test/fixtures/node_modules/mock-module/index.js
```

I would slacken the specificity of this

```suggestion
!packages/plugin-node-surrounding-code/test/fixtures/node_modules/
```

or even better can you put another `.dockerignore` in that package?